subsys/nrf_security/src/drivers/cracen/cracenpsa/include/cracen_psa_primitives.h

```diff
@@ -206,6 +215,7 @@ struct cracen_mac_operation_s {
 			struct sxmac ctx;
 			struct sxkeyref keyref;
 			uint8_t key_buffer[CRACEN_MAX_AES_KEY_SIZE];
+			cracen_cmac_context_t sw_ctx;
```

Comment:
This also needs to be conditionally included as well since it will increase stack usage for all the targets otherwise.